torch/distributions/laplace.py

```diff
@@ -55,5 +55,13 @@ def log_prob(self, value):
         self._validate_log_prob_arg(value)
         return -torch.log(2 * self.scale) - torch.abs(value - self.loc) / self.scale
 
+    def cdf(self, value):
```

Comment:
No `.icdf()`?

Reply:
Laplace's .cdf is a piecewise function. I was doubtful about adding an inverse, and later realized that the inverse could be piecewise as well. Will update as well.